stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -46,13 +48,13 @@
     private ExpiryDateEditText mExpiryDateEditText;
     private LockableHorizontalScrollView mScrollView;
     private View mCardNumberSpace;
+    private boolean mCardNumberIsViewed;
     private @ColorInt int mErrorColorInt;
+    private boolean mIsAmex;
+    private boolean mInitializedFlag;
     private int mScrollViewWidth;
     private int mScrollToPostion;
     private @ColorInt int mTintColorInt;
-    private boolean mCardNumberIsViewed;
-    private boolean mIsAmex = false;
```

Comment:
It does seem like a language quirk that not everyone might be aware of. Making it explicit would be helpful.